content/doc/book/blueocean/getting-started.adoc

```diff
@@ -17,156 +17,119 @@ endif::[]
 
 = Getting started with Blue Ocean
 
-This section describes how to get started with Blue Ocean in Jenkins. It
-includes instructions for link:#installing-blue-ocean[setting up Blue Ocean] on
-your Jenkins instance, as well as how to
-link:#accessing-blue-ocean[access the Blue Ocean UI] and
-link:#switching-to-the-classic-ui[return to the Jenkins classic UI].
+This section describes how to get started with Blue Ocean in Jenkins.
+It includes instructions for link:#installing-blue-ocean[setting up Blue Ocean] on
+your Jenkins instance, how to link:#accessing-blue-ocean[access the Blue Ocean UI], and link:#switching-to-the-classic-ui[returning to the Jenkins classic UI].
 
+include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
 == Installing Blue Ocean
 
-Blue Ocean can be installed using the following methods:
+You can install Blue Ocean using the following methods:
 
-* As a suite of plugins on an
-  link:#on-an-existing-jenkins-instance[existing Jenkins instance]  
+* As a suite of plugins on an link:#on-an-existing-jenkins-instance[existing Jenkins instance]. 
 * As a part of link:#as-part-of-jenkins-in-docker[Jenkins in Docker].
 
-
 === On an existing Jenkins instance
 
-When Jenkins is installed on most platforms, the
-plugin:blueocean[Blue Ocean plugin] and all its other dependent plugins (which
-form the Blue Ocean "suite of plugins") are not installed by default.
+When Jenkins is installed on most platforms, the plugin:blueocean[Blue Ocean] plugin and all necessary dependent plugins, which compile the Blue Ocean suite of plugins, are not installed by default.
 
-To install the Blue Ocean suite of plugins on an existing Jenkins instance, your
-Jenkins instance must be running Jenkins 2.7.x or later.
+To install the Blue Ocean suite of plugins on an existing Jenkins instance, your instance must be running Jenkins 2.7.x or later.
```

Comment:
While the statement is accurate, I don't think it is relevant.  Jenkins 2.7.x is so old that users won't be reading this page to learn about  it.
```suggestion
```

Reply:
Totally makes sense Mark, I'll remove this so that there's less ambiguity with the text!